static/sass/_patterns_social.scss

```diff
@@ -0,0 +1,44 @@
+@mixin p-social {
+  .p-social__heading {
```

Comment:
Please use the undocumented `p-muted-heading` for this

Reply:
I found it (by searching for "uppercase" in codebase), but it still needed different bottom spacing and color.

But if we introduce `.p-social-list` as you suggested below it, we could define spacing for `.p-social-list` and ignore color I guess...

Reply:
yeah I'd rather use the existing standard headings as used on www.canonical.com, then if we're not happy with the colour we should lobby to get p-muted-heading modified.